### DISCUSSION TITLE
Installation issue

### DISCUSSION THREAD

**Discussion by ghost** [coin-or/Couenne#64] [General]
Hi, I have been trying to install Couenne and ipopt on ubuntu 20.04 for the last couple of days. I could not install it properly. The instruction for the installation link on the [page](https://github.com/coin-or/Couenne) directs it to a[ page](https://github.com/coin-or/BuildTools/) which is currently unavailable.

 Kindly help me to complete the installation of the solvers in ubuntu 20.04.

**Comment by svigerske** [comment]
Well, the installation instructions for Couenne are very outdated. They still assume the old svn setup, which we migrated away from some years ago. In addition, someone seems to have setup a redirection from projects.coin-or.org to github.com/coin-or, which is wrong for BuildTools and some others, forbids access to stuff that wasn't migrated to GitHub so far, and hinders you to build Couenne from svn as in the old days (which wouldn't be so bad, since Couenne didn't really change since it is on GitHub).

The link to BuildTools won't really help you. But you can find this page now at https://github.com/coin-or-tools/BuildTools/blob/stable/0.5/docs/user-download.md#obtaining-the-code-using-subversion

The current suggested read about building a project like Couenne for new users can be found at https://coin-or.github.io/user_introduction#building-from-source

**Reply by tkralphs** [reply]
Yes, sorry, I have not gotten to updating the README for Couenne since we moved to Github. One of these days...

Another useful link is to [coinbrew](https://github.com/coin-or/coinbrew), which is the current recommended way to build (this is of course also mentioned in the detailed documentation @svigerske pointed you to).  On Ubuntu, coinbrew should just work out of the box.
```
coinbrew fetch Couenne@0.5.8
coinbrew build Couenne
```
